learn/platform/networks/moonriver.md

```diff
@@ -16,7 +16,7 @@ description: Moonriver（Moonbeam在Kusama上部署的平行链）的当前配
 目前，Moonriver具有以下配置：
 
 - 以平行链的形式运行并连接至Kusama中继链
-- 拥有一个由PureStake代表Moonbeam基金会运营并含有{{ networks.moonriver.staking.max_candidates }}位收集人的有效集。
```

Comment:
Here you removed PureStake but did not add OpsLayer? I mean I don't speak Chinese but unsure haha

Reply:
It was outdated. You can reference the Moonbeam page and see what I mean or the Moonriver page on the English site